### PR TITLE
Spacing breakpoints

### PR DESCRIPTION
**Pull request by AlecRaeside**

### CODE REVIEW

File: utilities/_u-spacing.scss [hunk 1]
```diff
@@ -32,23 +32,54 @@
...
-
-$u-spacing-small:   $spacing-half !default;
-
-$u-spacing-large:   $spacing-base-plus-half !default;
-
-$u-spacing-huge:    $spacing-double !default;
-
-
+// Apply at these breakpoints
+$spacing-breakpoints:   $default-utility-breakpoints !default;
```
Comment (chris-pearce) [inline utilities/_u-spacing.scss]: This needs `u-` at the start.

**Comment by kllevin** [comment]
LGTM

File: utilities/_u-spacing.scss [hunk 2]
```diff
@@ -32,23 +32,54 @@
...
-
-$u-spacing-large:   $spacing-base-plus-half !default;
-
-$u-spacing-huge:    $spacing-double !default;
-
-
+// Apply at these breakpoints
+$u-spacing-breakpoints:   $default-utility-breakpoints !default;
+
+$u-spacing-base:        $spacing-base !default;
```
Comment (chris-pearce) [inline utilities/_u-spacing.scss:42]: Can you comment each setting section?

File: utilities/_u-spacing.scss [hunk 3]
```diff
@@ -32,23 +32,54 @@
...
-
-$u-spacing-small:   $spacing-half !default;
-
-$u-spacing-large:   $spacing-base-plus-half !default;
-
-$u-spacing-huge:    $spacing-double !default;
-
-
+// Apply at these breakpoints
+$u-spacing-breakpoints:   $default-utility-breakpoints !default;
```
Comment (chris-pearce) [inline utilities/_u-spacing.scss]: Indention is off

**Pull request merged**